roles/common/tasks/main.yml

```diff
@@ -36,13 +32,17 @@
     dest=/home/jenkins/apache-maven-3.2.5-bin.tar.gz
     mode=0640
 
+- file: path=/home/jenkins/apache-maven-3.2.5-bin.tar.gz owner=jenkins group=jenkins mode=0644
```

Comment:
it's not redundant, the _fetch_ command instructs how to download it, the _file_ is an assertion on the final state. Keep in mind it's not procedural